fitters/include/DAF.h

```diff
@@ -111,13 +115,33 @@ class DAF : public AbsKalmanFitter {
    */
   void setAnnealingScheme(double bStart, double bFinal, unsigned int nSteps, unsigned int minIter, unsigned int  maxIter);
 
+  /**
+   * @brief Set the maximum number of iterations of the DAF.
+   */
   void setMaxIterations(unsigned int n) override {maxIterations_ = n; betas_.resize(maxIterations_,betas_.back());}
 
-  //! If all weights change less than delta between two iterations, the fit is regarded as converged.
+  /**
+   * @brief If all weights change less than delta between two iterations, the fit is regarded as converged.
+   */
   void setConvergenceDeltaWeight(double delta) {deltaWeight_ = delta;}
 
+  /**
+   * @brief Get a pointer to the internal Kalman fitter.
+   */
   AbsKalmanFitter* getKalman() const {return kalman_.get();}
 
+  /**
+   * @brief Set the maximum number of iterations of the internal Kalman fitter.
+   *
+   * Set the maximum number of iterations of the internal Kalman fitter.
+   * Note that the internal Kalman fitter can be called multiple times for each DAF iteration,
+   * up to the (maximum) number of iterations set by this method.
+   */
+  void setMaxIterationsKalman(unsigned int n) {getKalman()->setMaxIterations(n);}
```

Comment:
Here I added the additional setter we discussed (even though it is possible setting the maximum number of iteration of the Kalman via `DAF::getKalman()->setMaxIterations(n)` ).

Reply:
Should we set a default value for `n` here?